### PR TITLE
Added first round of named pawn and spell cards

### PR DESCRIPTION
**Pull request by KBgood**

### CODE REVIEW

File: Assets/Data/Cards/pawns/menotar.asset [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: bef7bbcd37c75064dad81e5f407169e2, type: 3}
+  m_Name: menotar
+  m_EditorClassIdentifier: 
+  cardArt: {fileID: 0}
+  cardName: Menotar
```
Comment (Stralor) [inline Assets/Data/Cards/pawns/menotar.asset:15]: "Menotar" the angry MRA version of the already 'roided Minotaur

File: Assets/Data/Cards/pawns/romanLegion.asset [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: bef7bbcd37c75064dad81e5f407169e2, type: 3}
+  m_Name: romanLegion
+  m_EditorClassIdentifier: 
+  cardArt: {fileID: 0}
+  cardName: Roman Legion
```
Comment (Stralor) [inline Assets/Data/Cards/pawns/romanLegion.asset:15]: Is this thematically consistent?

Comment (KBgood) [reply Assets/Data/Cards/pawns/romanLegion.asset:15]: welll.... I saw it as the gods manipulating the world including ours but it likely would depend on what the other decks might look like

File: Assets/Data/Cards/pawns/menotar.asset (hunk 1 above)
Comment (KBgood) [reply Assets/Data/Cards/pawns/menotar.asset:15]: hmmm makes me think of Men In Tights. So now Im thinking of Minotars in tights which I think is perfect. ;)

**Comment by KBgood** [comment]
I'm closing it yay!

**Pull request merged**